.github/ISSUE_TEMPLATE/bug-report-form.yml

```diff
@@ -0,0 +1,47 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what happend!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Language Version
+      description: What language are you using? please select Python > 3.6 if this is a bug relating to building the book.
+      options:
+        - 
+        - Python <= 3.6
+        - Python > 3.6
+        - R <= 3.5.1
```

Comment:
I think this form looks good! I'm wondering if we're keeping it as general as possible if it might be better to specify R<=3.6? I know we have 3.5.1 in DAP and hopefully there shouldn't be anyone using a version that old anywhere else unless they *really* have to. 3.6 is where all the big changes to CRAN requirements that mess up our packages came in and there are technically a couple of versions after 3.5.1 that would also have the same issues. Very unlikely to be a problem though so I'm happy if you think it would make more sense to keep it as it is!